var/spack/repos/builtin/packages/py-tpot/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyTpot(PythonPackage):
+    """
+    A Python Automated Machine Learning tool that optimizes machine learning
+    pipelines using genetic programming.
+    """
+
+    homepage = "http://epistasislab.github.io/tpot/"
+    url      = "https://files.pythonhosted.org/packages/6e/71/b4e01dff9b22eff990fca6483ab342edd480c8050f39df0143d27f423e4d/TPOT-0.11.5.tar.gz"
```

Comment:
https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#url

If you have trouble finding the correct URL, note that capitalization is important.

Reply:
It seems that is the case only in the last bit of the url but not the other parts. For example, the following link works:
```
https://pypi.io/packages/source/t/tpot/TPOT-0.11.5.tar.gz
```

Reply:
speaking of url of python packages. I think it would be a huge help to come up with a hack so that 'pythonhosted' links are automatically converted to 'pypi' links. I kind of looked into it at some point. Should be easy. Do you think it is a good idea? If so what would be a good place in spack libs to start from?

Reply:
Commented on your issue.